lib/pf/accounting.pm

```diff
@@ -74,6 +75,8 @@ Readonly our $ACCOUNTING_TRIGGER_RE => qr/
 Readonly our $DIRECTION_IN => 'IN';
 Readonly our $DIRECTION_OUT => 'OUT';
 
+our $CACHE = pf::CHI->new(namespace => "accounting");
```

Comment:
Wrap creating the cache in a function.
This will cause problems in a threaded environment